scripts/check_frontend_coverage.py

```diff
@@ -96,7 +96,6 @@
     'create-activity-button.directive.ts',
     'csrf-token.service.ts',
     'current-interaction.service.ts',
-    'drag-and-drop-html-string-editor.directive.ts',
```

Comment:
Note:

```drag-and-drop-html-string-editor.directive.ts``` was updated to ```translatable-html-content-id.component.ts```, ```list-of-sets-of-html-strings-editor.directive.ts``` to ```list-of-sets-of-translatable-html-content-id-editor.component.ts```, and ```set-of-html-string-editor.directive.ts``` to ```set-of-translatable-html-content-id-editor.component.ts```. The new editors are extremely similar to the old ones. Since the old editors did not have frontend tests, and I am a little crunched on time, I've replaced the old editors here with the new ones. @marianazangrossi I hope this is okay. I can update the starter issue to add frontend tests to include these new files once this PR is merged.